src/test/java/teammates/test/driver/HtmlHelper.java

```diff
@@ -385,6 +389,8 @@ private static String suppressVariationsInInjectedValues(String content) {
     private static String replaceUnpredictableValuesWithPlaceholders(String content) {
         Date now = new Date();
         SimpleDateFormat sdf = new SimpleDateFormat("EEE, dd MMM yyyy, ");
+        //Get session's time zone from content.
```

Comment:
You should mention in the comment that this assumes that the page only shows one timezone as it is possible for a page to show multiple time zones (eg. InstructorSearchPage) and this method would not work for such a page.